api/sbom.proto

```diff
@@ -153,3 +153,26 @@ message NodeList {
     repeated Edge edges = 2;
     repeated string root_elements = 3;
 }
+
+enum HashAlgorithm {
```

Comment:
Cool, this was exactly the idea, just one detail: these should be used in the actual hash information (eg lines 30 & 126). I'll merge it as is and fix as it is non breaking and I'll fix it in a followup